src/components/features/Login/Login.jsx

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable no-unused-vars */
+import clsx from 'clsx';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import AsideMenu from 'components/Layout/AsideMenu/AsideMenu';
+import { useAuthContext } from 'context/AuthContext.jsx';
+import useInput from 'hooks/useInput.jsx';
+import { isEmail, isPassword } from 'utils/validation';
+
+// Import styles:
+import './Login.scss';
+
+const Login = () => {
+  const { currentUser, isUserLoggedIn, handleLogin } = useAuthContext();
+  const [formIsValid, setFormIsValid] = useState(false);
+
+  const {
+    value: enteredEmail,
+    isValid: enteredEmailIsValid,
+    hasError: emailInputHasError,
+    handleInputChange: handleEmailInputChange,
+    handleInputBlur: handleEmailInputBlur,
+    reset: resetEmailInput,
+  } = useInput(isEmail);
+
+  const {
+    value: enteredPassword,
+    isValid: enteredPasswordIsValid,
+    hasError: passwordInputHasError,
+    handleInputChange: handlePasswordInputChange,
+    handleInputBlur: handlePasswordInputBlur,
+    reset: resetPasswordInput,
+  } = useInput(isPassword);
+
+  if (enteredEmailIsValid && enteredPasswordIsValid && !formIsValid) {
+    setFormIsValid(true);
+  } else if ((!enteredEmailIsValid || !enteredPasswordIsValid) && formIsValid) {
+    setFormIsValid(false);
+  }
+
+  // Check and set classes:
+  const emailInputClasses = clsx('login__form-control', emailInputHasError && 'invalid');
+  const passwordInputClasses = clsx('login__form-control', passwordInputHasError && 'invalid');
+
+  // Handle Submit:
+  const handleLoginSubbmission = async (e) => {
+    e.preventDefault();
+    await handleLogin(enteredEmail, enteredPassword);
+    resetEmailInput();
+    resetPasswordInput();
+  };
+
+  if (isUserLoggedIn) {
```

Comment:
If user is logged in and tries to open this page, we don't want to render the Login page at all - we want to redirect to `/pending-approval` (we decide that page is going to be the start page for users when they log in - but keep in mind we'll change this in the near future, when we handle multiple user roles - each role will have their own start page)